plugin-k8s/cmd/main.go

```diff
@@ -82,6 +82,10 @@ func (p *pluginServer) Scale(part string, time int64) (rec int32, err error) {
 	return a.Scale(time)
 }
 
+func (p *pluginServer) VerticalRecommendation(part string, time int64) (rec []*proto.RecommendedPodResources, err error) {
+	panic("unimplemented")
+}
```

Comment:
Just to confirm my expectations of the design, the Skenario plugin dispatch mechanism you are going to implement next will respect the capabilities the plugin reports at initialization.  Therefore this will not be necessary in the future, right?  You will be able to add a new method *without* requiring all existing plugins to implement that new method, right?

Reply:
Nevermind.  You are modifying the boilerplate golang code that goes with the plugin proto that each individual plugin (HPA and VPA will import).